cartography/intel/gcp/apigateway.py

```diff
@@ -79,6 +75,17 @@ def get_apigateway_locations(apigateway: Resource, project_id: str, common_job_p
         else:
             raise
 
+@timeit
+def transform_locations(locations: List[Dict], project_id: str) -> List[Dict]:
+    locations = []
+    for location in locations:
+        location['project_id'] = project_id
+        location['id'] = location['locationId']
+        location['consolelink'] = gcp_console_link.get_console_link(project_id=project_id, resource_name='apigateway')
```

Comment:
this `resource_name` seems inaccurate, it should be `api_gateway`. Please check in the cloudconsolelink project before using them.

Reply:
I have added the the consolelink implementeation for new resources and raised a PR